test/concurrify.test.js

```diff
@@ -1,11 +1,142 @@
 'use strict';
 
 var concurrify = require('../');
+var expect = require('chai').expect;
+var Z = require('sanctuary-type-classes');
+var type = require('sanctuary-type-identifiers');
+var FL = require('fantasy-land');
+
+var $$type = '@@type';
+
+var Identity = function(x){
+  var id = {x: x, constructor: Identity};
+  id[FL.ap] = function(mf){ return Identity(mf.x(x)) };
+  id[FL.map] = function(f){ return Identity(f(x)) };
+  return id;
+};
+
+Identity[FL.of] = Identity;
+Identity[$$type] = 'my/Identity';
+
+function mockAlt(a, b){ return b }
+function mockZero(){ return Identity('zero') }
```

Comment:
This is not the most lawful `zero` I've seen, but it simplifies unit testing. I only have to assert that our `mockZero` is used in the expected ways.